extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/AdditionalFieldConfig.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.logging.json.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Post additional fields. E.g. `fieldName1=value1,fieldName2=value2`.
+ */
+@ConfigGroup
+public class AdditionalFieldConfig {
+    /**
+     * Additional field value.
+     */
+    @ConfigItem
+    public String value;
+
+    /**
+     * Additional field type specification.
+     * Supported types: string, int, long.
+     * String is the default if not specified.
+     */
+    @ConfigItem(defaultValue = "string")
+    public String type;
```

Comment:
Normally you'd want to use an `enum` here when there are a limited number of valid values.

Reply:
Yup, that's better, mapped to an `enum`